### ISSUE TITLE
When creating a group, a SCAMP user must be assigned as the administrator

### ISSUE THREAD

**Issue by brentstineman** [SimpleCloudManagerProject/SCAMP#74]
When the group is created, a SCAMP user must be assigned as the administrator. 

Part of #43, dependent on #48

**Comment by brentstineman** [comment]
Required APIs would include... 

Get current Group Admins -> GET /api/groups/:groupId/admins
 returns: [ { "name" : "", "userId": "" } ]

Finder user in AD -> GET api/users/ADbyUPN/{upn}
 returns: { "name" : "", "upn" : "upn", "tenantID": "" }

Create SCAMP User -> POST api/users/
 payload: { "upn" : "", "name" : "", "tenantID" : "", "orgID" : "" }
 returns: userId (scamp User ID)

Add group admin -> POST /api/groups/:groupId/admins/:userId

Remove group manager -> DELETE /api/groups/:groupId/admins/:userId

**Comment by brentstineman** [comment]
key item remaining on this task is the ability to find and select a user from Azure AD to assign as an admin.

**Comment by vjrantal** [comment]
I'd propose closing this one since the basics are in place. There is still work, for example, in the deletion use cases, but we have a separate issue on those. @brentstineman What do you think?
